app/frontend/components/domains/permit-application/index.tsx

```diff
@@ -38,19 +38,26 @@ export const PermitApplicationIndexScreen = observer(({}: IPermitApplicationInde
     hasResetableFilters,
   } = permitApplicationStore
 
+  const { currentSandboxId } = sandboxStore
+
   const query = useQuery()
 
   const requirementTemplateId = query.get("requirementTemplateId")
   const templateVersionId = query.get("templateVersionId")
 
-  useSearch(permitApplicationStore, [requirementTemplateId || "", templateVersionId || ""])
+  useSearch(permitApplicationStore, [
+    requirementTemplateId || "",
+    templateVersionId || "",
+    JSON.stringify(currentSandboxId),
+  ])
   useFlashQueryParam()
   const resetQueryParams = useResetQueryParams()
 
   return (
     <Flex as="main" direction="column" w="full" bg="greys.white" pb="24">
       <PermitApplicationStatusTabs />
       <BlueTitleBar title={t("permitApplication.indexTitle")} />
+      {currentSandboxId}
```

Comment:
is this intentional or was it for testing?

Reply:
testing, good catch